File: Lista2/Exercícios_python/Sala/7.py
a = int(input("Digite um número inteiro: "))
if a==0:
    print("0 não possui módulo, uma vez que é um número neutro")
elif a>0:
    print("O módulo de", a, "é", a)
else:
    print("O módulo de", a, "é", a*(-1))

File: Lista2/Exercícios_C/Sala/7.c
#include <stdio.h>
int main ()

{
    int a;
    printf("Digite um número inteiro:");
    scanf("%i", &a);

    if (a==0) {
        printf("0 não possui módulo, uma vez que é um número neutro");
    }
    else if (a>0) {
        printf("O módulo de %.i é %i", a, a);
    }
    else {
        printf("O módulo de %i é %i", a, a*(-1));
    }
}


